scheduler/src/cook/components.clj

```diff
@@ -356,6 +357,8 @@
                                               fenzo-floor-iterations-before-warn)
      :fenzo-floor-iterations-before-reset (fnk [[:config [:scheduler {fenzo-floor-iterations-before-reset 1000}]]]
                                                fenzo-floor-iterations-before-reset)
+     :fenzo-fitness-calculator (fnk [[:config [:scheduler {fenzo-fitness-calculator "com.netflix.fenzo.plugins.BinPackingFitnessCalculators/cpuMemBinPacker"}]]]
```

Comment:
`com.netflix.fenzo.plugins.BinPackingFitnessCalculators/cpuMemBinPacker` is duplicated in multiple locations, maybe it should be extracted into a constant?